src/ray/protobuf/gcs_service.proto

```diff
@@ -139,6 +139,12 @@ message GetActorCheckpointIDReply {
   ActorCheckpointIdData checkpoint_id_data = 2;
 }
 
+// Service for placement group info access.
+service PlacementGroupInfoGcsService{
+  // Create placement group via gcs service
+  rpc CreatePlacementGroup(CreatePlacementGroupRequest) returns (CreatePlacementGroupReply);
```

Comment:
How do we check the status of a placement group? Or is that for a later PR?

Reply:
I will add the check in next pr. It looks like an actor creation in the core worker callback.